che-theia-factory/src/browser/factory-manager.ts

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
FYI license there https://github.com/eclipse/che-theia-factory-extension/blob/master/LICENSE is EPL 2.0

so copyright headers should be 2.0 as well.